R/harvest.R

```diff
@@ -0,0 +1,200 @@
+#' Harvest Adults Function
+#'
+#' This function calculates the total adult salmon harvest based on various parameters.
+#' This function is called within the larger model.R file before stray rates and prespawn survival is applied.
+#'
+#' @param adult_df A data frame containing information about adult salmon.
+#' @param year The target year for the harvest calculation.
+#' @param ocean_harvest_percentage The percentage of harvest from the ocean, same for all tributaries.
+#' @param tributary_harvest_percentage The percentage of harvest from tributaries, can vary by tributary.
+#' @param restrict_harvest_to_hatchery A logical vector indicating whether to restrict harvest to hatcheries for each tributary.
+#' @param no_cohort_harvest_years A vector of years where no cohort harvest is allowed.
+#' @param intelligent_crr_harvest A logical vector indicating whether to apply intelligent harvest based on Coho Retention Rate (CRR).
+#'
+#' @return A numeric value representing the total adult salmon harvest.
+#'
+#' @export
+
+harvest_adults <- function(adult_df,
```

Comment:
I wonder if we even need this since we plan to only add incidental take for winter run

Reply:
same goes for spring run. Lets talk about it!